src/components/Filters/SchemaSieve.ts

```diff
@@ -44,21 +42,27 @@ export const createFullTextSearchFilter = (
 	schema: JSONSchema,
 	term: string,
 ) => {
-	const stringKeys = reduce(
-		schema.properties,
-		(carry, item: JSONSchema, key) => {
+	const items = Object.entries(schema.properties as JSONSchema).reduce(
+		(itemsAccumulator, entry) => {
+			const [key, item] = entry;
 			if (
 				item.type === 'string' ||
-				(Array.isArray(item.type) && includes(item.type, 'string'))
+				item.type === 'number' ||
+				(Array.isArray(item.type) && item.type.includes('string'))
```

Comment:
I think we generally don't use arrays for `type`, so we can drop this for now, and add it if necessary. That being said, you'd have to check both `number` and `string` for consistency.

Reply:
this change was made 7 months ago by Lucian, I think it's used in jellyfish.
https://github.com/balena-io-modules/rendition/commit/ac125b02fa665429313cda8e33701b0ea06f0ac3

For now I would add the number check if you agree.

Reply:
@JSReds oh I didn't notice it existed, so yes just adding number is enough.